.eslintignore

```diff
@@ -1,3 +1,5 @@
+assets/js/jquery/3.5.1/jquery.min.js
```

Comment:
I think this is some merge conflict gone wrong, it should be removed as of https://github.com/18F/identity-site/pull/1153

```suggestion
```